src/index.ts

```diff
@@ -4,10 +4,17 @@ import { IntegrationEnv } from './env'
 import { ConfigStore } from 'fastly:config-store'
 import { returnHttpResponse } from './utils/returnHttpResponse'
 import { createFallbackErrorResponse } from './utils'
+import { setClientIp } from './utils/clientIp'
 
 addEventListener('fetch', (event) => event.respondWith(handleRequest(event)))
 
 export async function handleRequest(event: FetchEvent): Promise<Response> {
+  setClientIp(
+    event.client.address ??
+      event.request.headers.get('Fastly-Client-IP') ??
+      event.request.headers.get('X-Forwarded-For') ??
```

Comment:
I think we should remove `XFF` header because it is spoofable. `Fastly-Client-IP` looks to be true client IP according to [here](https://www.fastly.com/documentation/reference/http/http-headers/Fastly-Client-IP/) so that seems fine but I'd also like to see if it is possible spoof this header as well. Could you explain why you decided to use this header instead of [this usage](https://github.com/fingerprintjs/fingerprint-pro-fastly-proxy-integration-example/blob/main/src/index.ts#L13) we had before? I don't remember how we came up with this implementation before but would like to know why you picked the header over this @Orkuncakilkaya

Reply:
I applied this solution as a failover if one of them fails, next one should work. In order, our old way works and probably second condition will never work. But if a glitch happens we will have a failover scenario that won't affect identification. I can simply remove those and apply the old way but I think it's best to leave as it is because without a glitch in the system, it'll never work.

Reply:
In that case it is better to let it fail than picking an un-safe value. I'd remove any failovers

Reply:
Done 👍